application/server/connect/connectApiExpress.js

```diff
@@ -89,14 +90,22 @@ module.exports = function(app){
     let ret = await req.connectService.loadInstitutions(req.query.search_name || req.query.routing_number);
     res.send(ret);
   })
-  app.get('/jobs/:guid', async (req, res) => {
-    // this doesn't seem to affect anything as long as there is a response
-    res.send({
-      job: {
-        guid: req.params.guid,
-        job_type: 0, // must
-      }
-    })
+  app.get('/jobs/:member_guid', async (req, res) => {
+    if (['mx_int', 'mx'].includes(req.context.provider)) {
+      if (req.params.member_guid == 'null') {
+         res.send({ job: { guid: 'none', job_type: 0 }})
+         return;
+        }
+      let ret = await req.connectService.loadMemberByGuid(req.params.member_guid);
+      res.send(ret);
+    } else {
+      res.send({
+        job: {
+          guid: req.params.guid,
+          job_type: 0, // must
```

Comment:
I think it would be a good idea to store this required job_type field in an object at the top of the request handler, and spread it to both places that use it